examples/demo-app/src/actions.js

```diff
@@ -377,7 +377,14 @@ export function exportFileToCloud(handlerName = 'dropbox') {
 }
 
 export function setCloudLoginSuccess() {
```

Comment:
after a successful login it will also start the uploading without user interaction